templates/App/Field/_field_prototype.html.twig

```diff
@@ -12,6 +12,11 @@
             'data-action': 'change->field#updateChoiceList'
         }}) }}
     </div>
+    
+    <div>
```

Comment:
When adding fields to a template, the visibility field is very narrow:
![image](https://github.com/museum-engineering-project/koillectionPG/assets/86925036/bde18c2f-7fb5-4b3d-a2bf-bc9218a73533)
I think it would be better to make it wider, e.g. by adding `class = "col s3"` to the `div`.

Reply:
Done